transactions.md

```diff
@@ -104,15 +105,15 @@ The CPFP output value is adjusted depending on the actual transaction size.
 - outputs[1]:
     - value: N/A (may be many of them)
     - scriptPubkey: N/A
-- outputs[N]:
+- outputs[n]:
```

Comment:
Why ? :)

Reply:
A reader may be confused by the other usage of N for the N stakeholders.